vignettes/Import_fcs.Rmd

```diff
@@ -1,6 +1,10 @@
 ---
 title: "Importing and Basics of working with FCS files"
-output: html_notebook
+output: rmarkdown::html_vignette
```

Comment:
Need to use `html_vignette` for these to compile properly

Reply:
Thanks Andrew, will make this changes. I started thinking about it in terms of a notebook (hoping that it could be reused for cytoverse down the line).